mapviz_plugins/src/marker_plugin.cpp

```diff
@@ -405,6 +417,18 @@ namespace mapviz_plugins
 
   void MarkerPlugin::Draw(double x, double y, double scale)
   {
+    for (size_t i = 0; i < ui_.nsList->count(); i++)
```

Comment:
Any reason for this to not be `for (const auto& item : ui_.nsList)`?

Reply:
Nevermind that, I checked the class and discovered that it doesn't actually provide access to its container of items.  Lame.